src/functionalTest/java/uk/gov/hmcts/reform/prl/controllers/CallbackControllerFT.java

```diff
@@ -439,9 +317,7 @@ public void testAttachScanDocsWaChange() throws Exception {
             .post("/attach-scan-docs/about-to-submit")
             .then()
             .body("data.manageDocumentsRestrictedFlag", equalTo(TRUE),
-                  "data.manageDocumentsTriggeredBy", equalTo(STAFF))
-            .extract()
-            .as(AboutToStartOrSubmitCallbackResponse.class);
+                  "data.manageDocumentsTriggeredBy", equalTo(STAFF));
```

Comment:
It should be bulk scan and not cout staff as scanning will be done by bulk scan service